evm/p2p/utils.py

```diff
@@ -0,0 +1,4 @@
+def sxor(s1, s2):
+    if len(s1) != len(s2):
```

Comment:
Since this is used in cryptographic operations, should this use a constant time compare?

Reply:
Isn't that the case already, since we're comparing their lengths?

Reply:
I should not review code late at night. 👍